systemtest/playlist.xml

```diff
@@ -121,9 +119,6 @@
 		<levels>
 			<level>extended</level>
 		</levels>
-		<groups>
-			<group>system</group>
-		</groups>
```

Comment:
Any reason to remove the group system. Will this work?

Reply:
It was a mistake. Fixed!